ntbs-service/Startup.cs

```diff
@@ -312,6 +313,8 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
                 {
                     options.MessageTemplate =
                         "HTTP {RequestMethod} {RequestPath} responded {StatusCode} in {Elapsed:0.0000} ms ({RequestId})";
+                    // Set Logging Level of Request logging to Information - prevents duplicated exceptions in sentry. 
+                    options.GetLevel = (context, d, arg3) => LogEventLevel.Information; 
```

Comment:
```suggestion
                    options.GetLevel = (_, _, _) => LogEventLevel.Information; 
```
is this legal syntax and an option, or am I mixing languages here? :P